array-map/main.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(x => x * 2);
+console.log(doubled);
+
+const prices = numbers.map(x => Number.parseFloat(x).toFixed(2));
```

Comment:
Your currency strings need a dollar sign (`$`). Also, the numbers are already numbers, so `parseFloat` is unnecessary.

---

![image](https://user-images.githubusercontent.com/16143771/186994420-d692b705-6c1e-4500-823d-b2c7d0967ab4.png)